tests/testthat/test_isfiletext.R

```diff
@@ -0,0 +1,71 @@
+context("correctly identifying files as text vs binary")
+require("datasets")
+
+txtformats <- c("arff","csv","csv2","dump","fwf","psv","r","tsv","txt")
+binformats <- c("dbf","dta","rda","rdata","rds","sas7bdat","sav","xlsx","xpt")
+names(iris) <- gsub("\\.","_",names(iris))
+
+test_that("Required text formats recognized as text", {
+  for(xx in txtformats) expect_true(isfiletext(export(iris,
```

Comment:
style nit:

```
for (xx in txtformats) {
    ...
}
```

Reply:
Fixed all the ones I could find. Sorry, it's been a while and I forgot about the style guidelines, was not deliberately ignoring them.